cli/command/argParsingHelpers.go

```diff
@@ -0,0 +1,19 @@
+package command
+
+import "github.com/urfave/cli/v2"
+
+func prependArgParsingToAction(c *cli.Command, argName string, argGetter func(ctx *cli.Context) (string, error)) cli.ActionFunc {
```

Comment:
I see the point but this function makes it harder to read and understand the code. So I suggest you drop it.

Reply:
Fair. I thought there should be a helper for that but I agree that it ended up being very verbose and somewhat complicated. Removed it in https://github.com/taubyte/dreamland/pull/33/commits/5d616c3c577011a0b3b844af27372783e5010185